artemis/src/main/java/tech/pegasys/artemis/datastructures/beaconchainstate/ValidatorRecord.java

```diff
@@ -13,10 +13,15 @@
 
 package tech.pegasys.artemis.datastructures.beaconchainstate;
 
+import org.web3j.abi.datatypes.generated.Uint64;
```

Comment:
We should keep org.web3j.abi.datatypes.generated.Uint64 limited to only the pow package, right?